tests/unit/s2n_ktls_test.c

```diff
@@ -13,8 +13,18 @@
  * permissions and limitations under the License.
  */
 
+#include "tls/s2n_ktls.h"
+
+#include <sys/socket.h>
+
 #include "crypto/s2n_cipher.h"
 #include "s2n_test.h"
+#include "testlib/s2n_testlib.h"
+#include "tls/s2n_handshake_type.h"
+
+S2N_RESULT s2n_ktls_validate(struct s2n_connection *conn);
+S2N_RESULT s2n_ktls_validate_socket_mode(struct s2n_connection *conn, s2n_ktls_mode ktls_mode);
+S2N_RESULT s2n_ktls_retrieve_file_descriptor(struct s2n_connection *conn, s2n_ktls_mode ktls_mode, int *fd);
```

Comment:
It looks like some CI jobs are breaking because of this. You might want to put these in s2n_ktls.h.